crates/sui-indexer/src/store/pg_indexer_store.rs

```diff
@@ -1224,9 +1224,9 @@ impl<T: R2D2Connection + 'static> PgIndexerStore<T> {
         read_only_blocking!(&self.blocking_cp, |conn| {
             checkpoints::table
                 .filter(checkpoints::epoch.eq(epoch as i64))
-                .select(max(checkpoints::network_total_transactions))
-                .first::<Option<i64>>(conn)
-                .map(|o| o.unwrap_or(0))
+                .select(checkpoints::network_total_transactions)
+                .order_by(checkpoints::sequence_number.desc())
+                .first::<i64>(conn)
```

Comment:
Does it not need `limit(1)`? Not sure how sql compilation works

Reply:
no need as we do `first` here, which is basically limit 1 per diesel definition
```
    fn first<'query, U>(self, conn: &mut Conn) -> QueryResult<U>
    where
        Self: methods::LimitDsl,
        Limit<Self>: LoadQuery<'query, Conn, U>,
    {
        methods::LimitDsl::limit(self, 1).get_result(conn)
    }
 ```